cli/borges/consumer.go

```diff
@@ -53,9 +56,24 @@ func (c *consumerCmd) Execute(args []string) error {
 	wp.SetWorkerCount(c.WorkersCount)
 
 	ac := borges.NewConsumer(q, wp)
-	ac.Start()
 
-	return nil
+	var term = make(chan os.Signal)
+	var done = make(chan struct{})
+	go func() {
+		select {
+		case <-term:
+			log.Info("signal received, stopping...")
+			ac.Stop()
+		case <-done:
+		}
+	}()
+	signal.Notify(term, syscall.SIGTERM, os.Interrupt, os.Kill)
```

Comment:
If I understand this correctly, this is relaying to channel `term` the `SIGKILL` signal, but that signal must never be caught

Reply:
Ok, will remove `os.Kill`